.gitignore

```diff
@@ -25,3 +25,6 @@ cabal.project.local~
 *.skey
 var/
 .envrc
+my-atlas-config-maestro.json
```

Comment:
In local environment you can put these under `var` folder or perhaps `secrets` folder with `secrets` added to `.gitignore`.

Reply:
I would suggest omitting `my-preprod-maker-bot-config-gens-v2.yaml` etc from `.gitignore`.

Reply:
Done.  Moved all "secret" files to folder "secrets".